VideoWeb/VideoWeb/Controllers/AudioRecordingController.cs

```diff
@@ -55,6 +56,7 @@ public async Task<ActionResult> StopAudioRecordingAsync(Guid hearingId)
             try
             {
                 await _videoApiClient.DeleteAudioApplicationAsync(hearingId);
+                _logger.LogTrace($"Audio recording successfully stopped for hearingId: {hearingId}");
```

Comment:
shouldnt use string interpolation for logging, it wont add as custom property :)